packages/web/src/constants/common.constant.ts

```diff
@@ -1 +1,5 @@
 export const DEFAULT_NETWORK_ID = "dev.gnoswap";
+
+export const PATH = ["/earn"];
+export const PATH_10SECOND = ["/earn/pool/[pool-path]/remove", "/tokens/[token-path]"]; 
```

Comment:
Can you let me know the meaning of SECOND?

Reply:
i inherited the previous version of code base, it means that on the pages that represent by those paths the function have 10s time out for refetch interval

Reply:
it mean the `useGetPositionsByAddress` on those paths will continuously refetch after each 10 second

Reply:
By time, you meant data refetching cycles.
I'd appreciate it if you could do that in the comments as well.

Reply:
Yes, It a specific time for data refetching cycles, I will notice about the confusing code in the future.